src/action/find.rs

```diff
@@ -81,32 +84,9 @@ pub struct Find<'a, T: Send + Sync, Session = ImplicitSession> {
     session: Session,
 }
 
+#[option_setters_2(crate::coll::options::FindOptions)]
```

Comment:
The token stream is exported by creating a macro with a mangled name and the same visibility, so normal imports don't actually pick it up and you have to use the full path if you're not in the same module.